boards/common/arduino-atmega/include/board_common.h

```diff
@@ -63,6 +66,9 @@ extern "C" {
 #ifdef CPU_ATMEGA2560
 #define LED0_PIN            GPIO_PIN(1, 7)
 #define LED0_MASK           (1 << DDB7)
+#define LED0_ON             (PORTB |=  LED0_MASK)
+#define LED0_OFF            (PORTB &= ~LED0_MASK)
+#define LED0_TOGGLE         (PORTB ^=  LED0_MASK)
```

Comment:
This also was previously `PORTD`, but if I read the pinout correctly, this change is correct. @kfessel has a Arduino-Mega2560 on his desk, maybe he can quickly check?

Reply:
Hi, I have arduino-mega (chip clone) and provided fix works on it, too - I checked it once again now, after my last fixup. Both UNO and MEGA have LED on PORT #1 - so  PORTB in config is OK. PORTD was probably copied from Leonardo, where one LED is on PORT #3 - which means PORTD.

Reply:
thx for testing. Please squash!